package-toolkit/runtime/src/main/kotlin/com/atlan/pkg/Utils.kt

```diff
@@ -635,6 +635,12 @@ object Utils {
                         getInputFiles(sync, outputDirectory, prefix)
                     }
 
+                    "iam", "role" -> {
```

Comment:
Do these have to be just `iam` and `role`? Other object stores (besides S3) also support IAM and role-based auth — so if we at some point need to extend this in the future to support the same in those other cloud providers, just using `iam` or `role` is not going to be sufficiently distinct...

Reply:
@cmgrote That's what is in the Credentials file in `/tmp/credentials/success/result-0.json`. I don't have any control over that. I'd been more concerned with trying to change the configmap.